src/app/components/DeleteButton/DeleteButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+
+type DeleteButtonProps = {
+  handleDelete: () => void;
+  content: string;
+};
+
+export default function DeleteButton({
+  handleDelete,
+  content,
+}: DeleteButtonProps): JSX.Element {
+  return (
+    <Container>
+      <Delete onClick={handleDelete}>{content}</Delete>
+    </Container>
+  );
+}
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: var(--color-tertiary);
+  border-radius: 50%;
+  border: 1px solid var(--color-stroke);
+  padding: 0.625rem;
+  position: absolute;
```

Comment:
Not sure if I would make the whole component absolute. I'd rather leave this out here and when you integrate the component on the page, set the value there.